Klarna.Rest/Klarna.Rest/Models/Model.cs

```diff
@@ -40,8 +40,8 @@ public virtual string ConvertToJson()
                 Formatting.Indented,
                 new JsonSerializerSettings
                 {
-                    NullValueHandling = NullValueHandling.Ignore,
-                    DefaultValueHandling = DefaultValueHandling.Ignore,
+                    NullValueHandling = NullValueHandling.Include,
```

Comment:
This change is very dangerous. We don't need to send all null values to the server.
The **DefaultValueHandling.Include** is a good change and it was already merged to master.

Please, remove the  **NullValueHandling.Include** and I will be able to merge your change.